eng/versioning/version_client.txt

```diff
@@ -156,6 +156,7 @@ com.azure.spring:azure-spring-cloud-appconfiguration-config;2.5.0;2.6.0-beta.1
 com.azure.spring:azure-spring-cloud-feature-management-web;2.4.0;2.5.0-beta.1
 com.azure.spring:azure-spring-cloud-feature-management;2.4.0;2.5.0-beta.1
 com.azure.spring:azure-spring-cloud-starter-appconfiguration-config;2.5.0;2.6.0-beta.1
+com.azure.spring:spring-cloud-azure;4.0.0;4.1.0-beta.1
```

Comment:
`com.azure.spring:spring-cloud-azure` does not have latest released version, so I prefer this way:
```
com.azure.spring:spring-cloud-azure;4.1.0-beta.1;4.1.0-beta.1
```